system/ee/language/english/search_lang.php

```diff
@@ -152,6 +152,8 @@
 
     'today' => 'Today',
 
+    'expired_search_results' => 'Expired search. Try again.',
```

Comment:
I think this needs better wording, but I'd prefer native speaker to come forward here. @matthewjohns0n or @ops-andy any ideas?

(we also need to merge latest `6.dev` to make the tests pass)

Reply:
I like 'expired search' better than 'invalid action' which is definitely not helpful.  Not sure about the 'try again'.

Reply:
@robinsowell how about something similar to "The search results have expired. Please submit search form again." (although I'm not sure on part RE the form)

Reply:
I updated the message, but ignored the "submit form" part, because maybe there's no form involved.